app/helpers/schemes_helper.rb

```diff
@@ -26,4 +26,13 @@ def display_scheme_attributes(scheme)
     end
     base_attributes
   end
+
+  def scheme_availability(scheme)
+    availability = "Active from #{scheme.available_from.to_formatted_s(:govuk_date)}"
+    scheme.scheme_deactivation_periods.each do |deactivation|
+      availability << " to #{(deactivation.deactivation_date - 1.day).to_formatted_s(:govuk_date)}\nDeactivated on #{deactivation.deactivation_date.to_formatted_s(:govuk_date)}"
```

Comment:
I think we also need "Activating on..." in the case that the scheme is `:reactivating_soon` - happy to put this in later though for locations and schemes, or it could go in this PR, what do you think?

Reply:
Or is "Active from {future_activation_date}" enough do you think?

Reply:
Would that be a part of reactivation work?

Reply:
Sure, sounds good